src/main/java/com/passbook/sparkeighteen/service/UserService.java

```diff
@@ -0,0 +1,40 @@
+package com.passbook.sparkeighteen.service;
+
+import com.passbook.sparkeighteen.peristence.POJO.SignUpRequest;
+import com.passbook.sparkeighteen.peristence.POJO.SignUpResponse;
+import com.passbook.sparkeighteen.peristence.entity.UserEntity;
+import com.passbook.sparkeighteen.peristence.repository.UserRepository;
+import lombok.NonNull;
+import org.springframework.stereotype.Service;
+
+
+import java.util.Optional;
+
+@Service
```

Comment:
Why did we write @Service and not @Componnet

Reply:
- In an application, the business logic resides within the service layer so we use the @Service Annotation to indicate that a class belongs to that layer. 
- It is also a specialization of @Component Annotation like the @Repository Annotation. 
- One most important thing about the @Service Annotation is it can be applied only to classes. 
- It is used to mark the class as a service provider. 
- So overall @Service annotation is used with classes that provide some business functionalities. 
- Spring context will auto detect these classes when annotation-based configuration and class-path scanning is used.